### PR TITLE
integrated images and fit images to crop to fit

### LINKED ISSUE #58: Events List images are missing
## Type of Change
- [ ] New Feature Request
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary

During the completion of #24, it was realized images are not always available for events.  To support
1. Implement feature request in **website-lambda-meetup** requesting images get added to saved data response - https://github.com/ProvidenceGeeks/website-lambda-meetup/issues/11
1. Map that property to events object modeled and returned by Events API - https://github.com/ProvidenceGeeks/website-api-events/issues/16

Once those have been added and deployed, this issue can be implemented.

cc: @saimatsumoto

### PR DESCRIPTION
**Pull request by thescientist13**
## Related Issue

resolves #58 

## Summary of Changes

1. Got group images displaying
1. Depends on
    * https://github.com/ProvidenceGeeks/website-lambda-meetup/issues/11
    * https://github.com/ProvidenceGeeks/website-api-events/issues/16

**Desktop**
screen shot 2017-11-17 at 6 33 37 pm

**Mobile**
screen shot 2017-11-17 at 6 35 26 pm

### CODE REVIEW

File: src/components/card/card.scss [hunk 1]
```diff
@@ -1,9 +1,6 @@
 @import '../bootstrap/bootstrap';
 
-.card-img-overlay {
```
Comment (thescientist13) [inline src/components/card/card.scss]: couldn't see this class name being used in the `render` function

File: src/components/card/card.spec.jsx [hunk 2]
```diff
@@ -82,26 +82,25 @@ describe('Card component', () => {
...
+        'earbuds are OPTIONAL</p> <p>\\n\\n\\nProvidence Code Night with the Mayor Presented by IntraCity ' +
+        'Geeks.</p> <p>Send all questions to [masked]</p> <p>Introduction to Web Development!</p>' +
+        '<p>#ProvidenceCodeNight<br/>#MayorElorza<br/>#IntraCityGeeks</p>';
       const formattedContent = Card.formatHtmlContent(content);
 
       expect(formattedContent).toEqual(' Location: Providence Public Library 150 Empire Street, Providence, RI 02903- ' +
         'Rhode Island Room Requirements: 1. A laptop is required2. headphones or earbuds ar');
     });
+
+    it('should test formatHtmlContent when no value is provided', () => {
```
Comment (thescientist13) [inline src/components/card/card.spec.jsx]: just swapped the order of these tests and fixed the `it` block statement of behavior

**Comment by Peter-holyoke** [comment]
Looks like the location was removed from the date/time field in the card.

**Comment by thescientist13** [comment]
thanks @Peter-holyoke , good catch.  looking into it

**Comment by thescientist13** [comment]
@Peter-holyoke 
Not sure why I don't have the location in my screenshot, but I do see it locally.  That said, I see a couple issues with the Kenzan Hack Night tile
1. no image (I think there should be one)
1. no fallback placeholder

will try and account for both of these.

**Comment by thescientist13** [comment]
@Peter-holyoke 
Not sure why it was missing in my images, but it does show up locally, and all the tests are passing.

It seems props are passed down as strings, but passing undefined works, so I used a short circuit for that, and now the placeholder displays correctly.

Will reach out about getting an image for Kenzan's meetup

**Pull request merged**